pavelib/utils/db_utils.py

```diff
@@ -0,0 +1,203 @@
+"""
+Utility methods for bokchoy database manipulation.
+"""
+from __future__ import print_function
+import os
+import tarfile
+
+import boto
+from paver.easy import BuildFailure, sh
+
+from pavelib.prereqs import compute_fingerprint
+from pavelib.utils.envs import Env
+
+CACHE_FOLDER = 'common/test/db_cache'
+FINGERPRINT_FILEPATH = '{}/{}/bokchoy_migrations.sha1'.format(Env.REPO_ROOT, CACHE_FOLDER)
+
+
+def remove_files_from_folder(files, folder):
+    """
+    Remove the specified files from the folder.
+    Catch any errors as nonfatal.
+    """
+    for file_name in files:
+        file_with_path = os.path.join(folder, file_name)
+        try:
+            os.remove(file_with_path)
+            print('\tRemoved {}'.format(file_with_path))
+        except OSError:
+            print('\tCould not remove {}. Continuing.'.format(file_with_path))
+            continue
+
+
+def apply_migrations(db_cache_files, update_cache_files=True):
```

Comment:
nit: What do you think about changing this function name to be more reflective of what the underlying script is doing?

Reply:
I do agree with this also!